pkg/kp/rcstore/store.go

```diff
@@ -33,7 +33,10 @@ type Store interface {
 	// session that deletes its keys on invalidation.
 	Lock(id fields.ID, session string) (bool, error)
 
+	// Set the desired replica count for the given RC to the given integer.
 	SetDesiredReplicas(fields.ID, int) error
```

Comment:
I take it that you feel that both `Set` and `Add` are still necessary and worth keeping?